src/main/java/net/sf/jabref/logic/auxparser/AuxParserResult.java

```diff
@@ -50,6 +51,14 @@ public int getCrossRefEntriesCount() {
         return crossRefEntriesCount;
     }
 
+    public void countCrossRefEntry() {
+        crossRefEntriesCount++;
+    }
+
+    public void countNestedAuxFile() {
```

Comment:
I don't like the name. Maybe something more expressive as `increaseCountOfNestedAuxFiles()`?